waspc/data/Generator/templates/react-app/src/router.js

```diff
@@ -2,9 +2,9 @@
 import React from 'react'
 import { Route, BrowserRouter as Router } from 'react-router-dom'
 
-{=# routes =}
-import {= targetPage =} from './{= targetPage =}'
-{=/ routes =}
+{=# pagesToImport =}
+import {= importWhat =} from "{= importFrom =}"
```

Comment:
But this importWhat is not used anywhere? Below you are using targetPage. I guess those match, but is that coincidence, assumption, how does that happen?

Reply:
import what is actually the whole statement, not just page name. E.g. if the import was given as default, then it is just `PageName` (so we have `import PageName ...`, but if it was a named import then `importWhat` is `{ namedImport as PageName }` -> at the end we always want to have component under PageName name.f

I agree, it is not visible from the template and there is that connection that we don't enforce (at least in the template). I am not sure if we can make it more explicit -> consistency actually is enforced in at the generator lvl which produces data for the template. But since we don't have any logic available in the template not sure if we can avoid this. But good thinking.